src/test/java/api/loans/scenarios/OverrideRenewDeclaredLostItemTest.java

```diff
@@ -30,6 +32,17 @@ protected void performActionThatRequiresRefund(DateTime actionDate) {
     performActionThatRequiresRefund();
   }
 
+  @Before
+  public void setupLoanType() {
+    noteTypeFixture.generalNoteType();
+  }
```

Comment:
The name refers to a loan type yet the implementation refers to a note type, could this be renamed to remove this discrepancy?